### PR TITLE
Add sync service section

### PR DESCRIPTION
**Pull request by danielpassos**
This PR:

* Add a new services section with some useful links requested on AeroGear mailing
* Moves all services content (index, sidebar, menu and service landing page) to a YAML data file to avoid duplicated data
* Creates a new template for services landing page

Relates issues: 

* [AEROGEAR-7778](https://issues.jboss.org/browse/AEROGEAR-7778)

### CODE REVIEW

File: services/data-sync/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		<li><a href="https://github.com/aerogear/data-sync-ui">data-sync-ui</a></li>
+		<li><a href="https://github.com/aerogear/data-sync-server">data-sync-server</a></li>
+	</ul>
+
+	<h4>AeroGear Sync Client</h4>
+
+	<p>SDKs are WIP so, if you need to use this you will need to "install" it locally before move on</p>
+
+	<ul>
+		<li><a href="https://github.com/aerogear/aerogear-android-sdk/tree/sync/sync">aerogear-android-sdk</a></li>
```
Comment (wtrocki) [inline services/data-sync/index.html]: Wrong url => https://github.com/aerogear/aerogear-android-sdk/tree/master/sync

File: services/data-sync/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	<p>Both Server and UI have a Docker image if you want/need to run it isolated/outside of the Openshift README will help you on it. </p>
+
+	<ul>
+		<li><a href="https://github.com/aerogear/data-sync-ui">data-sync-ui</a></li>
+		<li><a href="https://github.com/aerogear/data-sync-server">data-sync-server</a></li>
+	</ul>
+
+	<h4>AeroGear Sync Client</h4>
+
+	<p>SDKs are WIP so, if you need to use this you will need to "install" it locally before move on</p>
```
Comment (wtrocki) [inline services/data-sync/index.html]: IMHO we should not put this information on the website. It looks bad.

File: services/data-sync/index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+	<ul>
+		<li><a href="https://github.com/aerogear/aerogear-android-sdk/tree/sync/sync">aerogear-android-sdk</a></li>
+		<li><a href="https://github.com/aerogear/aerogear-ios-sdk/tree/master/modules/sync">aerogear-ios-sdk</a></li>
+	</ul>
+
+	<h4>AeroGear Sync example apps</h4>
+
+	<ul>
+		<li><a href="https://github.com/aerogear/proposals/blob/master/dogfood.md">Proposal</a> </li>
```
Comment (wtrocki) [inline services/data-sync/index.html]: -1000 for proposal on the website

File: services/data-sync/index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+		<li><a href="https://github.com/aerogear/aerogear-android-example-apps/tree/master/memeolist">Android MemeOList</a></li>
```
Comment (wtrocki) [inline services/data-sync/index.html]: We will probably point people to tags rather than master in the future

**Comment by laurafitzgerald** [comment]
@danielpassos thanks for the pr. I agree with @wtrocki comments about not including certain items in this particular section. Now that I see this together with the other sections I wonder are these links better suited to be in the community section? It's a great list which gives someone who wants to contribute lots of context. Would you guys agree with putting it in the community section of the site?

**Comment by danielpassos** [comment]
Well, I'm going to close this so!

**Pull request closed without merging**